weavetest/abci.go

```diff
@@ -227,81 +222,3 @@ func numBlocks(totalTx, txPerBlock int) int {
 	}
 	return runs
 }
-
-var _ weave.ReadOnlyKVStore = (*WeaveRunner)(nil)
```

Comment:
But I use this in benchmarks.
I don't see where this functionality moved to?
#412 specifies moving, not removing, this functionality. Into apps. I am fine with the protobuf stuff gone and much functionality in weave/apps to remove import problems.

Reply:
I could not find code that is using it. All bnsd benchmarks are passing in this branch.

If you need the database access you can access it directly. Why should `WeaveRunner` implement a database interface to provide an indirect access?

Reply:
It was to easily reuse buckets to parse for queries
I see now that benchmarks never query nonce, but rather just increments a local counter, which removed the only query from the benchmarks.